elastic/integrations
# fix links in system dashboard

## What does this PR do?

This is a fix for #248

I swear, the dashboard links are cursed in Fleet.

## Checklist


## How to test this PR locally

- Build this integration via `mage build`, etc, start the elastic stack pointing at your test registry
- Make sure the system integration is installed
- Navigate the dashboards, select any of the system overview or host overview dashboards
- Go to the `System Navigation` header at the top right
- Select `System Overview` and make sure it actually works.
